File: simplevlcplayer/src/main/java/com/masterwok/simplevlcplayer/dagger/modules/FragmentModule.java
package com.masterwok.simplevlcplayer.dagger.modules;

import com.masterwok.simplevlcplayer.fragments.LocalPlayerFragment;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class FragmentModule {

    @ContributesAndroidInjector
    abstract LocalPlayerFragment contributeLocalPlayerFragment();

}

File: simplevlcplayer/src/main/java/com/masterwok/simplevlcplayer/dagger/modules/FragmentModule.kt
package com.masterwok.simplevlcplayer.dagger.modules

import com.masterwok.simplevlcplayer.fragments.*
import dagger.Module
import dagger.android.ContributesAndroidInjector


@Module
@Suppress("unused")
abstract class FragmentModule {

    @ContributesAndroidInjector
    internal abstract fun contributesLocalPlayerFragment(): LocalPlayerFragment

    @ContributesAndroidInjector
    internal abstract fun contributesRendererPlayerFragment(): CastPlayerFragment

    @ContributesAndroidInjector
    abstract fun contributesSubtitlesDialogFragment(): SubtitlesDialogFragment

    @ContributesAndroidInjector
    abstract fun contributesRendererItemDialogFragment(): RendererItemDialogFragment

}
